README.md

```diff
@@ -23,6 +23,25 @@ computers.
   </a>
 </div>
 
+## Example
```

Comment:
How about changing this to `Basic use` and removing the existing "Basic use" section at the bottom?